clients/privacy-center/cypress/fixtures/consent/experience_tcf_minimal.json

```diff
@@ -89,6 +89,221 @@
       "geolocation": {
         "location": "eea",
         "country": "eea"
+      },
+      "gvl": {
```

Comment:
automated preference saving was throwing an error because this was missing from the minimal experience mocks